.gitpod.yml

```diff
@@ -8,6 +8,8 @@ tasks:
   - before: export ACCESS_ENDPOINT=`gp url 4566`
     init: npm install
     command: /tmp/wait-for-it.sh localhost:4566 -t 0 -- ./init-localstack.sh && echo "Run npm run docker-dev to begin"
+  - init: npm install maildev -g
+    command: maildev --verbose
```

Comment:
Hang on, we can do that? If so, can you figure a way to supply the CMD arg to the docker image on local machine?